packages/cascara/src/private/ViewConfig/ViewConfigBase.js

```diff
@@ -56,6 +56,8 @@ const ViewConfig = ({
   const renderOptions = (options, isSelected) =>
     options.map((option) => {
       const { currentSelection, ...setterFunctions } = state;
+      // isEditable should not propapate to MemoViewConfigItem component
+      delete option.isEditable;
```

Comment:
@rociobaena I think what this is going to do is mutate the options that are being passed in to the MemoViewConfigItem. This means that the object being selected or removed will never be able to have isEditable on it. Basically we are never going to allow setting that property if we remove that value from the original object, which we do not want.

I think we may need to update the API of the ViewConfigItem to have an explicit `option` prop, instead of spreading the entire option into the component like we are doing now. That way we can always keep those props separate for use inside the ViewConfigItem.